src/index.ejs

```diff
@@ -81,14 +81,19 @@
     </div>
 
     <script>
-      const search = location.search
-      const params = search.split('?')[1].split('&').reduce((acc, i) => {
-        const tokens = i.split('=')
-        return {
-          ...acc,
-          [tokens[0]]: tokens[1]
-        }
-      }, {})
+      const params = new URLSearchParams(window.location.search)
+
+      // --- WORKAROUND ---
+      // `node-spellchecker` is loaded by `internal/modules/cjs/loader.js` before
+      // we can set the envoriment variable in our bootstrap routine.
+      const type = params.get('type')
+      const spellcheckerIsHunspell = params.get('slp') === '1'
+      if (spellcheckerIsHunspell && type !== 'settings') {
+        // Set option to always use Hunspell instead OS spell checker.
+        process.env['SPELLCHECKER_PREFER_HUNSPELL'] = 1 // eslint-disable-line dot-notation
```

Comment:
Why do we need to set `SPELLCHECKER_PREFER_HUNSPELL` environment variable? we can get this from `params` or `store.preferences`？

Reply:
`SPELLCHECKER_PREFER_HUNSPELL` is currently the only way to use Hunspell because the library doesn't have a function for this. We have to set the variable first, before `node-spellchecker` is loaded, due to the problem that `node-spellchecker` is loaded by `internal/modules/cjs/loader.js` before our application is started. I don't know why this happens because we require `node-spellchecker` in constructor and not via `import`.

Reply:
I understand, the `SPELLCHECKER_PREFER_HUNSPELL ` is required by `node-spellchecker`?